src/masternodeman.cpp

```diff
@@ -272,6 +273,7 @@ int CMasternodeMan::CheckAndRemove(bool forceExpiredRemoval)
             }
 
             it = mapMasternodes.erase(it);
+            LogPrint(BCLog::MASTERNODE, "Masternode removed. New total count: %d\n", mapMasternodes.size());
```

Comment:
would make more sense to log this outside of the loop only once

Reply:
Yes, sure.

Reply:
Moved the total count log outside the loop / squashed into last commit.

Reply:
cool, all good now.